### ISSUE TITLE
Fix the dark mode and focus change

### ISSUE THREAD

**Issue by kaal-coder** [CareerDevelopmentHub/Sudoku-Solver#17]
These are present in the code but not in a proper way, hence aren't visible on the website.
Fix this issue.

**Comment by DevDK12** [comment]
I would like to work on this

**Comment by DevDK12** [comment]
And also tell me what do you mean by focus change here . Could you explain it in detail more

**Comment by kaal-coder** [comment]
@DevDK12 On focus change, the title changes to - "Don't give up champ !"

**Comment by DevDK12** [comment]
> @DevDK12 On focus change, the title changes to - "Don't give up champ !"

You mean if focus of mouse is changed from sudoku box to outside of it then 'Sudoku Solver Project' should be changed to 'Don't give up champ !' ?

**Comment by kaal-coder** [comment]
Nope. If tab is changed to any other tab, then the title changes. [Refer - Stack Overflow ](https://stackoverflow.com/questions/66339765/changing-focus-for-html-accessibility)

**Comment by kaal-coder** [comment]
@DevDK12 make a PR if you have done. 
Make sure your branch is updated

**Comment by DevDK12** [comment]
> @DevDK12 make a PR if you have done. Make sure your branch is updated

I have already made sure everything was find and also made a PR request yesterday only. You should atleast review it

**Comment by kaal-coder** [comment]
@DevDK12 Check the reason why your PR was closed - https://github.com/CareerDevelopmentHub/Sudoku-Solver/pull/19#issuecomment-1282798253

**Comment by Pratik-k-sahoo** [comment]
I have solved the issue please can you assign me this issue?

**Comment by Pratik-k-sahoo** [comment]
Added the dark and light mode with icon change. 
Also added the focus change i.e when changes tab then document title changes and alert and notification saying "Don't Give Up Champ".
